### PR TITLE
changed date formatting. 

### PR DESCRIPTION
**Pull request by hadimarchi**
delivery date will simply be day of week if date is within the current week. If it is past that point the delivery date will be the day of the week followed by the date in mm/dd format

### CODE REVIEW

File: backend/app/order.py [hunk 1]
```diff
@@ -41,13 +42,29 @@ def place_order():
     return order_success()
 
 
+def is_next_week(delivery_date):
+    today = date.today()
+    return ((delivery_date - today) + timedelta(days=today.weekday())>=timedelta(days=7))
+
+
+def format_date(date_iso):
```
Comment (williamh890) [inline backend/app/order.py]: https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior

Comment (williamh890) [reply backend/app/order.py]: Plz

Comment (hadimarchi) [reply backend/app/order.py]: Me me me I'm Will and I'm soooo smart

Comment (williamh890) [reply backend/app/order.py]: lel

Comment (hadimarchi) [reply backend/app/order.py]: Changed to use strftime but not strptime so now we're both unhappy.

**Pull request merged**